connect.yaml

```diff
@@ -6,22 +6,31 @@ deployAs:
       standardConfiguration:
         - key: CTP_PROJECT_KEY
           description: Commercetools project key
+          required: true
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
+          required: true
         - key: CTP_SCOPE
           description: Commercetools client scope
+          required: true
         - key: CTP_AUTH_URL
           description: Commercetools Auth URL
+          required: true
         - key: CTP_API_URL
           description: Commercetools API URL
+          required: true
         - key: CTP_SESSION_URL
           description: Session API URL
+          required: true
         - key: CTP_JWKS_URL
           description: JWKs url
+          required: true
         - key: CTP_JWT_ISSUER
           description: JWT Issuer for jwt validation
+          required: true
       securedConfiguration:
         - key: CTP_CLIENT_SECRET
           description: Commercetools client secret
+          required: true
   - name: enabler
```

Comment:
Please move enabler entry before processor to clear cache during deployment as discussed this morning

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/171/commits/9070c015e2af5f71a21f37640635bcb18e0aef8d